### PR TITLE
Add log flume lift sound effect (when “chainlift” is enabled).

### PR DESCRIPTION
**Pull request by karst**
In preperation of #20258

### CODE REVIEW

**Review by tupaschoal — APPROVED** [review]
Seems sensible

**Comment by duncanspumpkin** [comment]
This will modify the game state you should bump the network version.

**Comment by karst** [comment]
Good catch

File: src/openrct2/network/NetworkBase.cpp [hunk 1]
```diff
@@ -46,7 +46,7 @@ using namespace OpenRCT2;
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
 
-#define NETWORK_STREAM_VERSION "1"
+#define NETWORK_STREAM_VERSION "2"
```
Comment (karst) [inline src/openrct2/network/NetworkBase.cpp:49]: @ZehMatt do you know why it's giving me a warning here?

Comment (tupaschoal) [reply src/openrct2/network/NetworkBase.cpp:49]: You touched on code where clang tidy has a recommendation. No need to change it, we can ignore and move on. Or if you want you can make another commit where you replace this network version macro by a constexpr.

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**